apps/src/code-studio/pd/international_opt_in/InternationalOptIn.jsx

```diff
@@ -71,7 +76,7 @@ class InternationalOptInComponent extends FormComponent {
    * @returns {boolean}
    */
   isColombiaSelected() {
-    return this.props.data?.schoolCountry?.toLowerCase() === 'colombia';
+    return this.props.data?.schoolCountry === 'colombia';
```

Comment:
These are only being removed because the schoolCountry prop is now in lowercase automatically.. but the removed line would also work, right?

Reply:
Before this PR, the ruby model was capitalizing the keys used for countries, therefore it was needed to lowercase `schoolCountry`, as the keys used in `international_opt_in_people.rb ` are lowercase.